src/components/pages/search-pages/teacher-search/TeacherSearchPage.styles.ts

```diff
@@ -0,0 +1,28 @@
+import { SxProps, Theme } from '@mui/material/styles';
+
+export const breadcrumbs: SxProps<Theme> = {
+  margin: '16px 0px 16px 0px',
+};
+
+export const layout: SxProps<Theme> = {
+  width: 1,
```

Comment:
Using `width: '100%` is common for creating responsive layouts that adapt to different screen sizes, as it ensures the element will always take up the full width of its container.
Plus you should keep same code style in project, and also you already used `width: '100%` in next styles